core/setup.py

```diff
@@ -71,7 +71,7 @@
         "sqlparse>=0.2.3,<0.5",
         # ----
         # These are major-version-0 packages also maintained by dbt-labs. Accept patches.
-        "dbt-extractor~=0.5.0",
+        "dbt-extractor~=0.5.1",
```

Comment:
the `~=` installs the latest patch for the `0.5` release so this should be installing `0.5.1` already.